ui/frontend/dist/modifiers.json

```diff
@@ -20,23 +20,42 @@
       "16-bit",
       "Anaglyph",
       "Anime",
+      "Art Nouveau",
```

Comment:
should probably move this out of the build, no reason for it to exist twice... figure that out later.